setup.py

```diff
@@ -68,6 +68,7 @@ def get_version(rel_path):
         "tqdm",
         "bokeh",
         "umap-learn",
+        "tensorflow-addons"
```

Comment:
See the note below, but I think we can remove this and use the blur function in the simclr module.